src/pages/landing/index.tsx

```diff
@@ -1,46 +1,74 @@
-import React from "react";
-import Icon from "@common/icons";
-import SignIn from "@components/landing/signIn";
+import React, { useEffect } from "react";
+
+import { Icon, Navbar } from "@components/common";
 import Phone from "@assets/images/i6.png";
 import Play from "@assets/images/play.png";
 
+import { Outlet, useLocation, useNavigate } from "react-router-dom";
+
+import { openMenu, closeMenu } from "@/state/features/menu/menuSlice";
+import { useAppDispatch, useAppSelector } from "@/state/app/hooks";
+
 import styles from "./Landing.module.css";
 
-const LandingContainer = () => (
-  <div className={styles.mainContainer}>
-    <div className={styles.leftContainer}>
-      <button className={styles.iconContainer} tabIndex={0}>
-        <Icon name="menu" />
-      </button>
-      <div className={styles.contentContainer}>
-        <Icon name="logo" />
-        <SignIn />
+const Landing = () => {
+  const dispatch = useAppDispatch();
+  const { menuOpen, activeLink } = useAppSelector((state) => state.menu);
+
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const handleMenu = (status: boolean) => {
+    status ? dispatch(openMenu()) : dispatch(closeMenu());
+  };
+
+  useEffect(() => {
+    if (location.pathname === "/") {
+      navigate("/sign-in");
```

Comment:
Same here with string. Maybe it would be better to define this different locations in some constants